### ISSUE TITLE
Create documentation showing how to write Ceramic-One events into Basin

### ISSUE THREAD

**Issue by mzkrasner** [recallnet/ipc#370]
- Using active testnet
- Will go in event-storage Hoku docs section

**Comment by stbrody** [comment]
Document how to configure ceramic-one to output document states to Recall (or any S3 compatible API).

**Comment by sanderpick** [comment]
@nathanielc just doubling checking this is on your plate? should it be moved into "in progress"?

**Comment by nathanielc** [comment]
Yes its on my plate, I plan to stat on it tomorrow, I have a few other bits in the works ATM.
